pca_ica/run_pca.m

```diff
@@ -33,7 +81,19 @@ function run_pca(movie_source, num_PCs)
 pca_info.movie_height = height;
 pca_info.movie_width  = width;
 pca_info.movie_frames = num_frames;
-pca_info.num_PCs = num_PCs; %#ok<STRNU>
-save(savename, 'pca_info', 'pca_filters', 'pca_traces', 'S');
+pca_info.num_PCs = num_PCs; 
+
+pca_info.is_trimmed = do_trim; 
+
+pca_info.is_medianfiltered = do_medfilt;  %#ok<*STRNU>
+if do_medfilt
+    pca_info.medfilt_halfwidth = medfilt_halfwidth;
+end

```

Comment:
Until we come up with a better ideal, I'd suggest a hierarchical storage scheme for `pca_info`, like:
```
pca_info.movie_height = height;
...
pca_info.num_PCs = num_PCs;

pca_info.trim.enabled = ...
pca_info.trim.idx_kept = ...

pca_info.medfilt.enabled = ...
pca_info.medfilt.halfwidth = ...
```

and also to use `idx_kept = ones(num_pixels,1)` (use column or row vector, to be consistent with the `idx_kept` returned by the `find` operation).

This way, we don't have to worry about different variables (e.g. `idx_kept`, in the current implementation) being present in the PCA `.mat` file.